packages/amesos2/src/Amesos2_Superludist_TypeMap.hpp

```diff
@@ -117,16 +117,24 @@ extern "C" {
   // Declare and specialize a std::binary_funtion class for
   // multiplication of SLUD types
   template <typename slu_scalar_t, typename slu_mag_t>
-  struct slu_mt_mult {};
+  struct slu_dist_mult {};
```

Comment:
thanks for the fixes and catching this, looks like a previous copy-paste mistake